bin/bot.py

```diff
@@ -5,11 +5,20 @@
 import json
 import asyncio
 from pymongo.errors import ServerSelectionTimeoutError
+import logging
+import logging.config
 
 from discord.ext import commands
 
 sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
 
+#Supresses other module logs
+logging.config.dictConfig({
+    'version': 1,
+    'disable_existing_loggers': True,
+})
+logging.basicConfig(filename=os.getenv("LOG_FILE"), level=logging.INFO, filemode='a', format='%(asctime)s - %(levelname)s - %(message)s')

```

Comment:
Does there need to be a `.env` file change along with this? If so, can that be reflected in [here](https://github.com/Givo29/everyBot/blob/devel/.env.example)?

Reply:
Done. I forgot to add it to the example env.